index.bs

```diff
@@ -1049,6 +1082,215 @@ Note: Invocations of {{MLContext/compute()}} will fail if any of the {{MLContext
 </details>
 </div>
 
+### {{MLContext/dispatch()}}  ### {#api-mlcontext-dispatch}
+
+Schedules the computational workload of a compiled {{MLGraph}} on the {{MLContext}}'s {{MLContext/[[timeline]]}}.
+
+<div dfn-for="MLContext/dispatch(graph, inputs, outputs)" dfn-type=argument>
+    **Arguments:**
+      - <dfn>graph</dfn>: an {{MLGraph}}. The computational graph to be executed.
+      - <dfn>inputs</dfn>: an {{MLNamedTensors}}. The inputs to the computational graph.
+      - <dfn>outputs</dfn>: an {{MLNamedTensors}}. The outputs of the computational graph.
+
+    **Returns:** {{undefined}}.
+</div>
+
+Note: `dispatch()` itself provides no signal that graph execution has completed. Rather, callers should await the results of reading back the output tensors. See [[#api-mlcontext-dispatch-examples]] below.
+
+<details open algorithm>
+  <summary>
+    The <dfn method for=MLContext>dispatch(|graph|, |inputs|, |outputs|)</dfn> method steps are:
+  </summary>
+    1. Let |allTensors| be a [=/list=] of {{MLTensor}}s consisting of |inputs|'s [=map/values=] [=list/extended=] by |outputs|'s [=map/values=].
+    1. If |allTensors| contains any duplicate [=list/items=], then [=exception/throw=] a {{TypeError}}.
+    1. [=list/For each=] |tensor| of |allTensors|:
+      1. If |tensor|.{{MLTensor/[[context]]}} is not [=this=], then [=exception/throw=] a {{TypeError}}.
```

Comment:
Not indented enough - doesn't render as a sub-list. Bikeshed's markdown needs 4 spaces.